relay-general/src/store/normalize.rs

```diff
@@ -293,15 +321,17 @@ fn remove_invalid_measurements(
         }
 
         // Retain payloads in _meta just for excessive custom measurements.
-        if let Some(measurement) = value.value_mut().take() {
```

Comment:
I don't understand the code that was there before, we already checked if value.value() returns the Some variant, why do it again to get value_mut when we could have done it in the start of the function? (which i changed it to do now)